internal/export/database/export.go

```diff
@@ -317,6 +317,8 @@ func (db *ExportDB) LookupSignatureInfos(ctx context.Context, ids []int64, valid
 				SignatureInfo
 			WHERE
 				id = any($1) AND (thru_timestamp is NULL OR thru_timestamp >= $2)
+			ORDER BY
+				id DESC
```

Comment:
The code comment says time order, but this is id. They are _probably_ the same, but not guaranteed to be.

Reply:
This is the best proxy we have as we don't capture create/effective times.

So relying on the serial column